doc/source/operations/upgrading-openstack.rst

```diff
@@ -35,6 +35,64 @@ Notable changes in the |current_release| Release
 There are many changes in the OpenStack |current_release| release described in
 the release notes for each project. Here are some notable ones.
 
+RabbitMQ SLURP upgrade
```

Comment:
This feels like it's in the wrong section (Notable changes in the |current_release| Release)

Reply:
Perhaps it should go into prerequisites?